validation-test/stdlib/Arrays.swift.gyb

```diff
@@ -158,8 +158,7 @@ ArrayTestSuite.test("${Kind}/append") {
 }
 % end
 
-ArrayTestSuite.test("ArraySlice/append") 
-  .xfail(.custom({true}, reason: "<rdar://problem/33997561>")).code {
+ArrayTestSuite.test("ArraySlice/append") {
```

Comment:
I'm still seeing the failure locally. There's still an internal bug tracking the crash. 

@gottesmm That sounds like the issue, I'll try an optimized build to confirm